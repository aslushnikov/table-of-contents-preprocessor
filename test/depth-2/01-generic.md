@@TOC@@

# Title

## 2nd level heading

Some text.

## Another 2nd level heading

Some other text here.
Actually multiline.


