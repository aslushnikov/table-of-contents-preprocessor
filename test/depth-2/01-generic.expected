- [Title](#title)
    - [2nd level heading](#2nd-level-heading)
    - [Another 2nd level heading](#another-2nd-level-heading)

# Title

## 2nd level heading

Some text.

## Another 2nd level heading

Some other text here.
Actually multiline.
